Mathlib/Data/ListM.lean

```diff
@@ -16,238 +16,241 @@ Lazy lists with "lazyness" controlled by an arbitrary monad.
 
 The inductive construction is unsafe (indeed, we can build infinite objects).
 This isn't so bad, as the typical use is with the tactic monad, in any case.
+Typical usage will be to do computations in `ListM`, and then eventually extract
+the first result (with `head` or `firstM`), first several results (using `take`),
+or all results (using `force`). At this point you can use the `unsafe` term elaborator
+from `Std.Util.TermUnsafe` to return to the "safe" world.
 
-As we're in meta anyway, we don't bother with proofs about these constructions.
+As we're unsafe anyway, we don't bother with proofs about these constructions.
 -/
 
-
 universe u v
 
 /-- A monadic lazy list, controlled by an arbitrary monad. -/
 unsafe inductive ListM (m : Type u → Type u) (α : Type u) : Type u
-  | nil : ListM m α
-  | cons : m (Option α × ListM m α) → ListM m α
+| nil : ListM m α
```

Comment:
I think mathport got it right, by indenting the constructors, didn't it?

Reply:
I guess so. Weird that we indent after `inductive`, but not after `match`, but okay! Fixed now.